README.md

```diff
@@ -1,2 +1,45 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
 # OOP-school-library
-a tool to record what books are in the library and who borrows them.
+> a tool to record what books are in the library and who borrows them.
+
+## Built With
+
+- Ruby
+
+
+## Getting Started
+
+To get a local copy follow these simple example steps.  
+
+1.- Open the terminal window and clone the repository using this command:  
+`git clone https://github.com/JuanLPalacios/OOP-school-library.git` 
+
+2.- Change the directory to the enumerables directory  
+`cd OOP-school-library`  
+3.- Install the project's dependencies by running this command:   
+`bundle install`  
+
+
+## Authors
+
+👤 **Juan Luis Palacios**
+
+- GitHub: [@JuanLPalacios](https://github.com/JuanLPalacios)
+- Twitter: [@JuanLuisPalac20](https://twitter.com/twitterhandle)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/juan-luis-palacios-p%C3%A9rez-95b39a228/)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## 📝 License
+
+This project is [MIT](./LICENSE.md) licensed.
```

Comment:
- [ ] The MIT link is broken, I recommend fixing that so your project is documented professionally.